src/config/routes.ts

```diff
@@ -0,0 +1,4 @@
+export enum ROUTES {
+  HOME = "/",
+  GDG = "/gdg-page",
```

Comment:
just `gdg` is fine, the actual route would be `gdg-page`, which is less user friendly imo